Archives/ClmFile.cpp

```diff
@@ -405,24 +390,57 @@ namespace Archives
 		return strippedExtensions;
 	}
 
+	WaveFormatEx ClmFile::CreateDefaultWaveFormat() {
+		WaveFormatEx waveFormat;
+
+		waveFormat.wFormatTag = 1; // WAVE_FORMAT_PCM
+		waveFormat.nChannels = 1; // mono
+		waveFormat.nSamplesPerSec = 22050; // 22.05KHz
+		waveFormat.nAvgBytesPerSec = 44100; // nSamplesPerSec *nBlockAlign
+		waveFormat.nBlockAlign = 2; // 2 bytes/sample = nChannels * wBitsPerSample / 8
+		waveFormat.wBitsPerSample = 16;
+		waveFormat.cbSize = 0;
+
+		return waveFormat;
+	}
```

Comment:
This struct can probably be a static class member using the struct initialization syntax:
```
static const WaveFormatEx DefaultWaveFormat = {1, 1, 22050, 44100, 2, 16, 0};
```
You can split that into multiple lines if you want to retain the per field comments.

Reply:
Sounds good.